.github/workflows/backport_v1.yaml

```diff
@@ -24,7 +24,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Backporting
-        uses: pierreprinetti/backport@v1
+        # https://github.com/kiegroup/git-backporting/releases/tag/v4.1.0
+        uses: kiegroup/git-backporting@8c010b43e4b0715886de96cd8ea819a8fa42ffd8
```

Comment:
The syntax you suggest is valid. However, I never use tags in Github actions because the Action author can later re-target the same tag to point to another commit and make our repo execute arbitrary code.

I actually think that accepting tags is a dangerous supply-chain attack vector that Github shouldn't have let in.

While in this case we know and trust the author of the action, pinning dependencies to a checksum is a good habit that I prefer not to deviate from.